### ISSUE TITLE
No warning by RB or BNG if rule defined without transformation

### ISSUE THREAD

**Issue by GoogleCodeExporter** [RuleWorld/bionetgen#48]
```
In rule #3 of the attached model a complexation is defined without bonding: 
P53(a,b,c) + P53(a,b,c) <-> P53(a,b,c).P53(a,b,c).  Therefore, this rule 
doesn't actually do anything.  No error or warning message is thrown and when 
run BNG simply ignores it, listing it in the 'reaction rules' block of the NET 
file with no defined transformations.  This sort of thing should be caught.  I 
cannot think of a scenario where a modeler would intentionally define a rule 
that does not encode a transformation.


```

Original issue reported on code.google.com by `lh64@cornell.edu` on 10 Aug 2012 at 10:54
